packages/server/src/sdk/app/views/index.ts

```diff
@@ -252,3 +252,49 @@ export function syncSchema(
 
   return view
 }
+
+export async function renameLinkedViews(table: Table, renaming: RenameColumn) {
+  const relatedTables: Record<string, Table> = {}
+
+  for (const field of Object.values(table.schema)) {
+    if (field.type !== FieldType.LINK) {
+      continue
+    }
+
+    relatedTables[field.tableId] ??= await sdk.tables.getTable(field.tableId)
+    break
+  }
+
+  for (const relatedTable of Object.values(relatedTables)) {
+    let toSave = false
+    const viewsV2 = Object.values(relatedTable.views || {}).filter(
+      sdk.views.isV2
+    )
+    if (!viewsV2) {
+      continue
+    }
+
+    for (const view of viewsV2) {
+      for (const relField of Object.keys(view.schema || {}).filter(f => {
+        const tableField = relatedTable.schema[f]
+        if (tableField.type !== FieldType.LINK) {
```

Comment:
Is `tableField` guaranteed to be non-undefined here? Think this might be a case where not having strict index checks is giving us a false sense of security.

Reply:
It should not be undefined, as there are other processes that cleanse old ones. But just in case, as technically it might be not the case, I will add a guard!